spec/getCountriesSpec.js

```diff
@@ -135,4 +138,46 @@ describe("The index", () => {
       ).toBeTrue();
     });
   });
+
+  describe("The getAllCountriesByLandlock", () => {
+    it("returns all the landlocked countries", () => {
+      expect(countryApi.getCountriesByLandLock(true).length).toEqual(45);
+    });
+
+    it("returns all the countries which are not landlocked", () => {
+      expect(countryApi.getCountriesByLandLock(false).length).toEqual(151);
+    });
+
+    it("returns countries with isLandLocked set to true", () => {
+      expect(countryApi.getCountriesByLandLock(true)[0]).toEqual({
+        country: "afghanistan",
+        capital: "kabul",
+        currency: "afghani",
+        native_language: ["dari persian", "pashto"],
+        famous_for: "rugs, taliban",
+        phone_code: "+93",
+        flag: "https://flagpedia.net/data/flags/h80/af.png",
+        drive_direction: "right",
+        is_landlocked: true
+      });
+    });
+
+    it("returns countries with isLandLocked set to false", () => {
+      expect(countryApi.getCountriesByLandLock(false)[0]).toEqual({
+        country: "albania",
+        capital: "tirane",
+        currency: "lek",
+        native_language: ["albanian"],
+        famous_for: "mother teresa",
+        phone_code: "+355",
+        flag: "https://flagpedia.net/data/flags/h80/al.png",
+        drive_direction: "right",
+        is_landlocked: false
+      });
+    });
+
+    it("should return undefined if passed any value other than boolean", () => {
+      expect(countryApi.getCountriesByLandLock(1).length).toEqual(undefined);

```

Comment:
From my point of view, there are two ways, how _API_ should behave in that case:
1. Throw an error that suggest what is the proper value
2. Return empty array

From a user perspective, 1. seems to be more handy, since it tells me what I did wrong and help me without checking the documentation.

Reply:
Yes i agree throwing an error would be suffice, will make the changes.